windowPreview.js

```diff
@@ -960,8 +960,8 @@ var Preview = Utils.defineClass({
     },
 
     _getBackgroundColor: function(offset, alpha) {
-        return 'background-color: ' + this._getRgbaColor(offset, alpha) + 
-               'transition-duration:' + this._previewMenu.panel.dynamicTransparency.animationDuration;
+        return 'background-color: ' + this._getRgbaColor(offset, alpha) +
+               'transition-duration:' + this._previewMenu.panel.dynamicTransparency.animationDuration + ';';
```

Comment:
`this._previewMenu.panel.dynamicTransparency.animationDuration` already ends with a semicolon, see [here](https://github.com/home-sweet-gnome/dash-to-panel/blob/master/transparency.js#L149).